Libraries/LibGUI/MultiView.cpp

```diff
@@ -106,33 +106,42 @@ MultiView::MultiView()
             on_drop(index, event);
     };
 #endif
-    set_view_mode(ViewMode::Icon);
 
     build_actions();
+    // `m_view_mode` already is ViewMode::Icon. See `set_view_mode`.
+    set_view_mode(ViewMode::Icon, Update::Force);
```

Comment:
Let's just start `m_view_mode` with `ViewMode::Invalid`. Then you won't need this awkward forced update logic :)

Reply:
I don't think it's awkward, but I changed it.